### ISSUE TITLE
Just Cause 3

### ISSUE THREAD

**Issue by JacksMafia** [x360ce/x360ce#574]
game is crashing on all of x64 versions don't know why i just can't play it with keyboard and mouse 
i know it uses xinput9_1_0 but can't get it to work 
any help?

**Comment by AmrKhedr** [comment]
Same Problem here, game crashes on start

**Comment by vzabawski** [comment]
Yes, same issue. Also, x360ce crashes when I'm enabling xinput 9.1 x64. First, I'm getting "Unnecessary files were found", after that I'm getting an exception "Permission denied" when the program tries to delete xinput1_3.dll.

**Comment by EvoJocys** [comment]
This report is about the 3.x line, where x360ce was an xinput DLL copied into each game's folder. The current 4.x line works differently: it creates a virtual Xbox 360 controller that every game sees, with nothing copied into game folders. Game-specific problems from the DLL era do not carry over.

The latest release is 4.21 (September 2026): https://github.com/x360ce/x360ce/releases

Closing as part of a tracker clean-up after the 4.21 release. If it still happens on 4.21, reopen this issue with the version and what you see.
